db/migrate/20181212110943_create_onboarding_records.rb

```diff
@@ -1,9 +1,8 @@
 class CreateOnboardingRecords < ActiveRecord::Migration[5.2]
   def change
-    enable_extension "pgcrypto" unless extension_enabled?("pgcrypto")
 
     create_table :school_onboardings do |t|
-      t.uuid :uuid, null: false, default: 'gen_random_uuid()'
```

Comment:
@jamesjefferies I've spent some un-fruitful time trying to get Travis to use PG 10 as this function doesn't exist on the version of PG that's running by default with no luck. Can never connect to the DB if I set a different version. And, I have no guarantee that this function is going to be available on production so I'm switching it out for a manual UUID for now. Unless you have any ideas?

Reply:
@jamesalmond oh that's a pain - this is the only thing I saw from a quick look, they do now support 10 apparently, maybe it needs to be forced to use a certain OS version somewhere... 
```
addons:
  postgresql: "10"
  apt:
    packages:
    - postgresql-10
    - postgresql-client-10
env:
  global:
  - PGPORT=5433
```

Reply:
I've tried various combinations of this but it didn't seem to work. Might be worth trying it in isolation.